src/test/java/org/broadinstitute/hellbender/tools/exome/HashedListTargetCollectionUnitTest.java

```diff
@@ -541,4 +542,43 @@ public void testRandomLookupExonByLocation() {
                         nonOverlappingExomeIntervals.get(1).getStart())}
         };
     }
+
+    @Test
+    public void testTrimmedInterval() {
+        final String CONTIG = "contig";
+        final String NONEXISTANT_CONTIG = "contig";
```

Comment:
Why is `CONTIG` the same as `NONEXISTANT_CONTIG`?  I think it's the position that's non-existent, no the contig...